### PR TITLE
Validation Bulk Fixes

### PR DESCRIPTION
**Pull request by mohitp930**

### CODE REVIEW

File: BingMaps/v8-web-control/modules/venue-map-module/venuemapoptions-object.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Venue Maps Options Object | Microsoft Docs"
+description: Learn about the Venue Maps options object.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/modules/venue-map-module/venuemapoptions-object.md:3]:
```suggestion
description: Learn about the Venue Maps options object including its properties and the VenueMapMetadataLoader Method.
```

**Review by stevemunk — CHANGES_REQUESTED** [review]

File: BingMaps/spatial-data-services/query-api/query-by-id.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Query by ID | Microsoft Docs"
+description: Learn hot to query by ID with the spatial data services query API.
```
Comment (stevemunk) [inline BingMaps/spatial-data-services/query-api/query-by-id.md:3]:
```suggestion
description: Describes how to search a data source for one or more entities by entity ID using the spatial data services query API.
```

File: BingMaps/v8-web-control/articles/cross-platform-bing-maps-v8-apps.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Cross Platform Bing Maps V8 apps | Microsoft Docs"
+description: Learn about Bing Maps V8 cross platform support.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/articles/cross-platform-bing-maps-v8-apps.md:3]:
```suggestion
description: Learn about Bing Maps V8 cross platform support that enables support on mobile devices that support HTML5.
```

**Review by stevemunk — CHANGES_REQUESTED** [review]

**Comment by mohitp930** [comment]
> @mohitp930 Please address the 2 requested changes and let me know when done and I will merge into main. Thanks!

Done and done. Thanks for all of your help.

**Pull request merged**